client/src/components/Review/DetailsPanel.js

```diff
@@ -108,23 +115,24 @@ class DetailsPanel extends React.Component {
             <STYLED_featuredReview>
```

Comment:
This div should have a semantic child tag such as an H4 or H3. Try not to leave text tags alone inside of a div.